nettacker/database/db.py

```
@@ -397,18 +853,46 @@ def logs_to_report_json(target):
+                            "port": json.loads(log[2]),
+                            "event": json.loads(log[3]),
+                            "json_event": json.loads(log[4]),
+                        }
+                        return_logs.append(data)
+                    return return_logs
+            finally:
+                try:
+                    cursor.close()
+                    connection.close()
+                except Exception:
+                    pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)